woocommerce/navigation
# Add onClick prop to navigation items

Adds an onClick prop to the navigation items so we can pass in click handlers.

### Testing

1. Add an `onClick` prop to any item.
1. Check that the click handler fires on click.
